gramps/plugins/webstuff/webstuff.py

```diff
@@ -178,12 +178,15 @@ def load_on_reg(dbstate, uistate, plugin):
          path_img_48x48("gramps-geo-mainmap.png"), None, [], [] ],
         ]
     # If we add css user files, we must restart gramps to use them.
-    list_files = os.listdir(USER_CSS)
-    for cssfile in list_files:
-        CSS_FILES.append([cssfile, 1, cssfile.replace('.css', ''),
-                          os.path.join(USER_CSS,cssfile),
-                          None, [], [] ])
-    return CSS_FILES
+    try:
+        list_files = os.listdir(USER_CSS)
+        for cssfile in list_files:
+            CSS_FILES.append([cssfile, 1, cssfile.replace('.css', ''),
+                              os.path.join(USER_CSS,cssfile),
+                              None, [], [] ])
+        return CSS_FILES
+    except:
```

Comment:
Well, you should catch only what you expect to see; `except FileNotFoundError:` or `except OSError:` or something like that.